distributions/dependencies.json

```diff
@@ -1,18 +1,14 @@
 {
-  "bedrock": [
+  "sambanova": [
```

Comment:
can we make these keys be lexicographically sorted? also not clear why sambanova doesnt want mcp

Reply:
I ran a json-sort on this file in my next rev. We may need to automate if we care about these dependency orderings.

Reply:
Sure thing. Automated the sort.